locust/runners.py

```diff
@@ -412,6 +418,9 @@ def worker(self):
                 logger.info("Got quit message from master, shutting down...")
                 self.stop()
                 self.greenlet.kill(block=True)
+            elif msg.type == "switch":
+                logger.info("Change locust file from master")
```

Comment:
Is it possible to put the file name also in this logger. I expect to have kind of message like `Got new test from master, test file switched to blablabla.py`